src/documents/InclusiveMakingAndHacking.md

```diff
@@ -57,7 +57,7 @@ We encourage those considering hacking and making in learning contexts to be awa
 
 ### Asking Who Is Excluded
 
-Hackathon events can unintentionally [exclude participants](WhatIsTheApproach.md#whoIsExcluded) through limitations in their design. How do we redesign our approach to running, promoting and recruiting for our events? What strategies can help us avoid inadvertent exclusion?
+Hackathon events can unintentionally [exclude participants](/WhatIsTheApproach.html#whoIsExcluded) through limitations in their design. How do we redesign our approach to running, promoting and recruiting for our events? What strategies can help us avoid inadvertent exclusion?
```

Comment:
In the md files we used to link to the other md documents so that the links would work when viewed directly in GitHub. We'd then re-write these in the build process to point to the generated HTML document. I take it this functionality has been removed. Is it possible to bring it back?

Reply:
Discussion [in Matrix](https://matrix.to/#/!JcoHDrfLedPQdFhAXn:matrix.org/$CwIQq71dbYAvXWdw37ZwDaKOx5pp6bQ7Xb8-mk8P71s?via=matrix.org).